process/project_proposals.adoc

```diff
@@ -10,30 +10,18 @@ image::project-stages.png[Project stages]
 
 === Sandbox process
 
-_Note: we are currently experimenting with a https://docs.google.com/document/d/1IlhqnmhSzs3aTPMkp_75JPYz6LD21aK1sNPrBNruDt0/edit?usp=sharing[new lighter-weight process for Sandbox proposals]. If you are interested in submitting a project for Sandbox we recommending reaching out to CNCF staff, or raising in the TOC channel on CNCF Slack, for advice on how to proceed in the short term._
+Note: We have changed the sandbox proposal process to a https://docs.google.com/document/d/1IlhqnmhSzs3aTPMkp_75JPYz6LD21aK1sNPrBNruDt0/edit?usp=sharing[new lighter-weight process for Sandbox proposals].
 
 All exceptions (and "no" outcomes) are handled by the TOC. Possible "no" outcomes include "not at this time", and the project may be encouraged to re-apply after addressing issues. Timeframes are approximate, to set expectations.
 
 image::sandbox-process.png[Sandbox process]
-
 . *Project Proposal*
-   * Project proposed through a https://github.com/cncf/toc/issues[GitHub issue]
-   * Template TBD
-. *TOC Triage* _2 weeks for a straightforward referral to a SIG; decline or defer could require more discussion_
-   * Performed by any TOC member
-   * Lazy TOC consensus before declining outright or deferring assessment at this time
-   * Picks appropriate https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md[CNCF SIG] to perform assessment
-. *SIG Assessment* _1-2 months_
-   * Project presents to SIG
-   * Lightweight - this is pre-DD
-   * Confirms that project accepts the https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#sandbox-stage[Sandbox entry requirements]
-   * SIG makes recommendation to TOC: Sandbox / Incubation / Decline 
+   * Project proposed through a https://docs.google.com/document/d/1IlhqnmhSzs3aTPMkp_75JPYz6LD21aK1sNPrBNruDt0/edit?usp=sharing[form]
+   * Project proposed for contribution through a https://github.com/cncf/toc/pulls[Pull request] 
```

Comment:
Please remove the pull request option as we're only doing form now. This is confusing.

Reply:
No, we'll need to have something to merge in at the end of the process -- we still need a pull request to merge into the repo.

Reply:
ahh that makes sense

Reply:
Ah that makes sense but I was not able to gather that from the bullets. Maybe numbering them would help?

Reply:
I am surprised about this, I also thought we were just doing the form. What's the content that we need for merging into the repo? And could this be populated after the vote if it's successful, to keep the application process lightweight?